### PR TITLE
chore: Add prosemirror-dev-tools as a dependency

### PR DESCRIPTION
**Pull request by jonathonherbert**
## What does this change?

At the moment, we use a CDN to source the prosemirror-dev-tools. Why do that, when you can include it as a npm package? 

This PR also tidies up the `index.html` a bit, and removes an `index.html` in the root of the project that as far as I can tell is redundant.

## How to test

Run `yarn` in the project root, and then run the project normally. You should still see the devtools icon in the right hand side of the page. Open it – the devtools should work as expected: 

tr-dev-tools

### CODE REVIEW

**Comment by ParisaTork** [comment]
So glad the owl is back!

Owl

Tested and can confirm this has been successful:

Typing

File: demo/style.css [hunk 1]
```diff
@@ -155,6 +155,341 @@ h6 {
...
+  position: relative;
+}
+
+.ProseMirror {
+  word-wrap: break-word;
+  white-space: pre-wrap;
+  white-space: break-spaces;
+  -webkit-font-variant-ligatures: none;
+  font-variant-ligatures: none;
+  font-feature-settings: "liga" 0; /* the above doesn't seem to work in Edge */
```
Comment (ParisaTork) [inline demo/style.css:168]: You're right to highlight issues of cross-browser compatibility, as this only works in 2 specific versions of Edge: https://caniuse.com/?search=font-feature-settings

File: demo/style.css [hunk 2]
```diff
@@ -155,6 +155,341 @@ h6 {
...
+  left: -32px;
+  right: -2px; top: -2px; bottom: -2px;
+  border: 2px solid #8cf;
+  pointer-events: none;
+}
+
+/* Protect against generic img rules */
+
+img.ProseMirror-separator {
+  display: inline !important;
```
Comment (ParisaTork) [inline demo/style.css:205]: Is !important absolutely essential here?

File: demo/style.css [hunk 3]
```diff
@@ -155,6 +155,341 @@ h6 {
...
+  font-size: 12px;
+}
+
+.ProseMirror-invalid {
+  background: #ffc;
+  border: 1px solid #cc7;
+  border-radius: 4px;
+  padding: 5px 10px;
+  position: absolute;
+  min-width: 10em;
```
Comment (ParisaTork) [inline demo/style.css:458]: I've seen a few examples of px and em units mixed together in this file. Would it be worth having a discussion around whether we should stick to relative units throughout?

**Comment by ParisaTork** [comment]
I've added a few tiny comments to open up some interesting conversations about some choices we currently make in CSS. This is purely driven by curiosity and congrats on this PR! 🎉

**Review by ParisaTork — APPROVED** [review]

File: demo/style.css (hunk 2 above)
Comment (jonathonherbert) [reply demo/style.css:205]: So sorry, Parisa, I took your comment as an approval! My bad.

This is all imported straight from Prosemirror (https://prosemirror.net/css/editor.css), and it's super low stakes.

Perhaps we could keep it in a separate file to make its provenance clearer? Happy to raise another PC.

Comment (ParisaTork) [reply demo/style.css:205]: I've approved all the changes, so you were right to do so! Thanks for clarifying here, this all looks good

File: demo/style.css (hunk 3 above)
Comment (ParisaTork) [reply demo/style.css:458]: ^Resolved given explanation above in another style.css review

**Pull request merged**